library/src/lib/tile/tile-grid/tile-grid.directive.ts

```diff
@@ -1,22 +1,54 @@
 import { Directive, ElementRef, Inject, Input } from '@angular/core';
 import { AbstractFdNgxClass } from '../../utils/abstract-fd-ngx-class';
 
+/**
+ * Directive that represents a tile grid. 
+ * A Tile Gird is a collection of fd-tile components in a gird layout. 
+ * ```html
+ * <fd-tile-grid [col]="3">
```

Comment:
I didn't know we were doing this, but directives should not be used as components :|

Out of the scope of this docs issue though so don't worry bout it